src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -829,6 +850,17 @@ private void AddRefReadonlyIfRequired()
             }
         }
 
+        private void AddReadOnlyIfRequired()
+        {
+            // 'readonly' in this context is effectively a 'ref' modifier
+            // because it affects whether the 'this' parameter is 'ref' or 'in'.
+            if (format.MemberOptions.IncludesOption(SymbolDisplayMemberOptions.IncludeRef))
```

Comment:
>IncludeRef [](start = 79, length = 10)

Could you update the xml doc for `SymbolDisplayMemberOptions.IncludeRef`?

Reply:
Done